Components/badges.css

```diff
@@ -2,41 +2,33 @@
     display: flex;
     flex-direction: row;
     padding: 1rem;
-    border: 1px solid var(--color-black);
+    border: 1px solid var(--color-gray-200);
     width: 60%;
 }
-
 .badge-icons{
     font-size: 3rem;
 }
-
 .badge-box{
     margin: 1rem;
     padding: 1rem; 
 }
-
 .badge{
-    color:var(--color-white);
-    background: var(--color-brown);
+    color:var(--color-gray-100);
+    background: var(--color-primary);
     border-radius: 50%;
     padding: 0.1rem 0.5rem;
```

Comment:
You can use padding: 2px 8px here, rem is not required as it wont make much difference

Reply:
Thanks, this one done too.